src/e2ee/sharedKeyManagement.ts

```diff
@@ -74,12 +83,14 @@ export const useRoomSharedKey = (roomId: string): string | undefined => {
 
 export const useIsRoomE2EE = (roomId: string): boolean | null => {
```

Comment:
Seeing `| null` on this return type makes me wonder what we were ever returning `null` for, and why we aren't doing that now

Reply:
Looks like we used to return null if we couldn't find the room: https://github.com/vector-im/element-call/blame/535712d108a3257501457ae24f497dc7d52ed671/src/e2ee/sharedKeyManagement.ts#L76 - unsure why that changed though.